src/mount/mount-tool.c

```diff
@@ -853,8 +853,12 @@ static int stop_mount(
         polkit_agent_open_if_enabled(arg_transport, arg_ask_password);
 
         r = sd_bus_call(bus, m, 0, &error, &reply);
-        if (r < 0)
-                return log_error_errno(r, "Failed to stop mount unit: %s", bus_error_message(&error, r));
+        if (r < 0) {
+                if (streq(suffix, ".automount") &&
+                    sd_bus_error_has_name(&error, "org.freedesktop.systemd1.NoSuchUnit"))
+                        return 0;
```

Comment:
Hmm, but why? Is there something special about automount units that a request to stop a non-existent unit needs to be silently ignored?

Reply:
`systemd-mount -u` unconditionally stops both mount and automount units. But, the automount unit may not exist, even if the specified path is a correct mount point. Showing the error may confuse users.

Reply:
Note that after 4a78b637808c5ba436dbbcaef54724be827298fd, loading automount unit fails if the corresponding unit file does not exist.